### DISCUSSION TITLE
Technically possible legal issue, github account was set up for me without my consent, and is stopping me progressing further in python course]

### DISCUSSION THREAD

**Discussion by c2995509** [community/community#72675] [New to GitHub]
### Select Topic Area

General

### Feature Area

Issues

### Body

GIthub account was set up without my consent via open university, unfortunately I know need to have a github account for a python course run by AI Core. HOw do I proceed?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @c2995509 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
